ideascube/db_router.py

```diff
@@ -1,30 +1,44 @@
+
+_databaseToUse = {
+    ('search', 'search') : 'transient'
+}
```

Comment:
> Why not simply get the model from the app_label and model_name?

I didn't get the model because we need a apps to use `apps.get_config()` and I think it was simpler to move the information here instead of found how to get the apps.
After investigation, it seems that get the apps is no complex at all (`from django.app import apps`)

However, the `apps` is the present apps and a model of migration may not exists at the time we apply the migration (the model name changed). If we attach the information to the model, we may loose the information.
We can default the database to use to 'default' and it will work most of the time (for now) but
it could lead to problem in the future (if we rename the search table in the transient db).

So I suggest to keep move with a separate dict of which db to use depending of a `app_label` and `model_name`